cockpit/src/subscriptions-register.jsx

```diff
@@ -29,6 +29,21 @@ import Select from "./Select/Select.jsx";
  *   - properties as in defaultRegisterDialogSettings()
  */
 class PatternDialogBody extends React.Component {
+    constructor(props) {
+        super(props);
+        // Bind this to non React method
+        this.cleanCredentialValues = this.cleanCredentialValues.bind(this);
+    }
+    cleanCredentialValues() {
+        console.debug('Cleaning credential values.');
+        this.props.onChange('user', '');
+        this.props.onChange('password', '');
+        this.props.onChange('activation_keys', '');
+        this.props.onChange('org', '');
```

Comment:
I think this logic would fit better in `index.js` (can add another object to the `Object.assign(registerDialogDetails)` line) ex: 
```js
Object.assign(registerDialogDetails, subscriptionsClient.config, {
  user: '',
  password: '',
  activation_keys: '',
  org: '',
});
```

In this way all the initialization of `registerDialogDetails` is done in one place.